src/server/chromium.ts

```diff
@@ -96,8 +99,8 @@ export class Chromium implements BrowserType {
     }
 
     const usePipe = chromeArguments.includes('--remote-debugging-pipe');
-    if (usePipe && webSocket)
-      throw new Error(`Argument "--remote-debugging-pipe" is not compatible with "webSocket" launch option.`);
+    if (usePipe && isServer)
```

Comment:
Vice versa is also a problem (passing --remote-debugging-port=9222 in args to launch()), since we only check `isServer` in line 134. I suggest to add a test for this.

Reply:
This set of checks is so confusing. I'm thinking of replacing it all with a throw if user passes `--remote-debugging-pipe` or `--remote-debugging-port`, wdyt?

Reply:
sgtm!